Doppler.Sap.Job/Database/IDopplerRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Doppler.Sap.Job.Service.Database
+{
+    public interface IDopplerRepository
+    {
+        public Task<IEnumerable<object>> GetBillingClientInformation();
```

Comment:
I would prefer a more descriptive type, even for initial steps.

Reply:
With @fgchaio we talk about it and I created a new `BillingCredit ` class in  this commit 
https://github.com/MakingSense/doppler-jobs/pull/6/commits/bfd4b80459426a2ca5d14619077e2202fb713732
Is possible that change this entity in the future